ReactAndroid/src/main/java/com/facebook/react/uimanager/TouchTargetHelper.java

```diff
@@ -144,8 +148,19 @@ private static boolean isTransformedTouchPointInView(
       localX = localXY[0];
       localY = localXY[1];
     }
-    if ((localX >= 0 && localX < (child.getRight() - child.getLeft()))
-        && (localY >= 0 && localY < (child.getBottom() - child.getTop()))) {
+    float hitSlopTop = 0;
```

Comment:
@ide @jesseruder I just wanted to point out that the behaviour will be different than what people are used to while developing on other platforms. The main difference is that in RN on native side we treat *all* views as touchable (all views can receive touch events and their tags will be passed to JS which to allow JS side to make the decision on which view should respond to the touch).

So normally when you'd use `TouchDelegate` on android for a button that is in a `LinearLayout` and the surrounding views are all not buttons (similarly to the example [1] where Child1 is a button and Child2 is not) you will be able to click the button by tapping in any part of the "extended" area.

With this approach you'll get a quite inconsistent behaviour when the hitSlop area will only extend the view in left and top direction but not in right and bottom.

With that said I still think there are use-cases where this simple approach will work just fine and save developers some time on wrapping their touchable views etc. E.g. when you already have margins that are greater or equal to hitSlop or when you have absolutely positioned view with enough space around it. So I think it would be useful to add a few lines of comments to the property definition to make this clear for people who might be using it in the future